website_sale_product_attribute_value_filter_existing/readme/USAGE.rst

```diff
@@ -0,0 +1,3 @@
+* Go to Website Shop
+* Active product attributes filter
```

Comment:
This has changed a bit in v16 as now is a theme panel option that is enabled on either top or bottom in the layout. I'd move it to CONFIGURATION.rst as well.

Reply:
yes, moved to `CONFIGURATION.rst` and update more details. thanks for pointing out